.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDatePeriodParserConfiguration.cs

```diff
@@ -207,6 +207,22 @@ private int ConvertChineseToInteger(string yearChsStr)
             var year = 0;
             var num = 0;
 
+            var regionTitleMatch = ChineseDateExtractorConfiguration.RegionTitleRegex.MatchExact(yearChsStr, trim: true);

```

Comment:
Ideally we should not have such duplicated code. Can't it be restructured to re-use at least a common method?